### PR TITLE
Initial Class Structure Implemented

### PR DESCRIPTION
**Pull request by mike-rus**
The general structure of classes has been created,
the basic entity ChatRegistry has been implemented,
functionality for self-destructing messages has been added, 
a basic task scheduler and adding/removing clubs.

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -1,19 +1,109 @@
...
 if __name__ == "__main__":
+
     application = ApplicationBuilder().token(BOT_TOKEN).build()
+    chat_registry.add_app_instance(application)
 
-    start_handler = CommandHandler("start", start)
-    application.add_handler(start_handler)
+    application.add_handler(CommandHandler("help", help))
+    application.add_handler(CommandHandler("start", start))
+    application.add_handler(CommandHandler("start", add_team))
```
Comment (polina-gritsenko) [inline src/main.py]: probably `application.add_handler(CommandHandler("add_team", add_team))`

File: src/main.py [hunk 2]
```diff
@@ -1,19 +1,109 @@
...
+    """
+    await chat_registry.add_club(
+        Club(update.message.chat_id, update.effective_chat.title)
+    )
+    await chat_registry.delete_message_delayed(
+        update.message.chat_id, update.message.message_id, delay=10
+    )
+
+
+async def start(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
```
Comment (polina-gritsenko) [inline src/main.py]: looks like a duplication of the above method

Comment (mike-rus) [reply src/main.py]: done

**Comment by mike-rus** [comment]
@polina-gritsenko have a look ones again please

**Review by polina-gritsenko — APPROVED** [review]

**Pull request merged**